src/css/fonts.css

```diff
@@ -0,0 +1,102 @@
+@import url("https://fonts.googleapis.com/css?family=Source+Sans+Pro:200,200i,300,300i,400,400i,600,600i,700,700i,900,900i");
```

Comment:
I'm thinking that we should remove the 200 font weight since even 300 isn't accessible for some font sizes (and super thin on some browsers).